### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by olawale-o**
This Pull Request contains:

- A contact-bg.png File in the assets Directory
- An index.html File with a section Tag of class contact-section
- A style.css File with styling for contact-section class

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -303,3 +303,78 @@ body {
...
+  background-color: #6070ff;
+  padding: 0;
+  border-top-left-radius: 125px;
+  width: 100%;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 50px;
+  justify-items: center;
+}
```
Comment (teekaytech) [inline style.css]: As I've mentioned earlier, you guys have done an amazing job. Your contact form works well and the UI is great :muscle: 

I realized that you've used the `Grid` layout style for this section. The grid layout works fine and it's acceptable. However, this project specifically stated that you should use CSS Flexbox layout. It is important we always stick to the project specification. Therefore, I suggest you replace the grid layout with flexbox.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__This is a brilliant effort from you guys. You've really done an amazing job so far. In this PR, here are my major highlights:__

- Correct configuration of linters with no errors reported :100: 
- Descriptive PR title, summary and commit messages :+1: 

__However, you need to make some required changes before moving on to the next assignment. Please check my comments below and fix them as appropriate.__

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
N/A


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: index.html [hunk 2]
```diff
@@ -1,249 +1,338 @@
...
+            <form action="https://formspree.io/f/mwkabeev" method="POST">
+              <fieldset class="fieldset">
+                <div class="field">
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    class="control"
+                    value="Olawale"
```
Comment (MahmoudBakr23) [inline index.html:297]: - [ ] **OPTIONAL** I suggest deleting the `value` attribute and just leave the `placeholder`

File: index.html [hunk 3]
```diff
@@ -1,249 +1,338 @@
...
-                        <li class="tag">javascript</li>
-                    </ul>
-                    <button class="btn-project">See Project</button>
+                <div class="field">
+                  <input
+                    type="email"
+                    name="email"
+                    id="email"
+                    class="control"
+                    value="olawale@micoverse.org"
```
Comment (MahmoudBakr23) [inline index.html:308]: - [ ] **OPTIONAL** I suggest deleting the `value` attribute and just leave the `placeholder`

**Review by MahmoudBakr23 — APPROVED** [review]
Hello Team, @olawale-o @hiromataba😃,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**